GoogleForCreators/web-stories-wp
# Improve Site Kit Google Analytics integration

## Summary

Removes code copied from Site Kit, and instead uses the `googlesitekit_amp_gtag_opt` filter to add custom event tracking for stories.

Depends on https://github.com/google/site-kit-wp/pull/1596.

## To-do

In #570, we'll have to deal with a non-Site Kit fallback.

## User-facing changes

N/A

## Testing Instructions

1. Check out https://github.com/google/site-kit-wp/pull/1596 and install on your site (or use Site Kit Tester plugin)
1. Make sure GA is connected
1. Check out this PR on your site
1. Verify that `amp-analytics` script & snippet are output correctly

Example:

<img width="926" alt="Screenshot 2020-05-28 at 13 33 38" src="https://user-images.githubusercontent.com/841956/83136517-e6e6db00-a0e7-11ea-9920-f559d7b84582.png">

---



Fixes #1168
